### ISSUE TITLE
Deleting backwards in a tabular deletes entire lines

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#185]
Place the caret after the D:
```
\begin{table}[ht]
\begin{tabular}{l|l}
AB & CD \\
\hline
EF & GH
\end{tabular}
\end{table}
```
and delete backwards once. This will result in
```
\begin{table}[ht]
\begin{tabular}{l|l}
\hline
EF & GH
\end{tabular}
\end{table}
```

**Comment by ktraunmueller** [comment]
**Fixed** in 0.9.19

**Comment by ktraunmueller** [comment]
And yes, _there's a unit test for that_ ™

<img width="580" alt="screen shot 2017-12-18 at 21 54 48" src="https://user-images.githubusercontent.com/650763/34127638-2af67a44-e43e-11e7-9bf8-8c645e852ca3.png">
